ondewo/vtsi/voip.proto

```diff
@@ -69,19 +69,27 @@ service VoipSessions {
     // start an ondewo-sip-sim instance that listens for calls with the given parameters
     rpc StartCallInstance (StartCallInstanceRequest) returns (StartCallInstanceResponse) {
         option (google.api.http) = {
-            post: "/v2/{session=projects/*/agent/sessions/*}:StartListener"
+            post: "/v2/{session=projects/*/agent/sessions/*}:StartCallInstance"
             body: "*"
         };
     }
 
     // stop/kill an ondewo-sip-sim listener instance
     rpc StopCallInstance (StopCallInstanceRequest) returns (StopCallInstanceResponse) {
       option (google.api.http) = {
-        post: "/v2/{session=projects/*/agent/sessions/*}:StopListener"
+        post: "/v2/{session=projects/*/agent/sessions/*}:StopCallInstance"
         body: "*"
       };
     }
 
+    // start an ondewo-sip-sim instance that listens for calls with the given parameters
```

Comment:
This can start both listeners and callers, right? Let's add this info to the comment.

Also the info that the only difference between caller and listener is that caller has a phone number specified. That might be also helpful as comment of `StartCallInstanceRequest`